code/modules/clothing/spacesuits/flightsuit.dm

```diff
@@ -795,35 +795,45 @@
 	if(istype(I, /obj/item/weapon/stock_parts))
 		var/obj/item/weapon/stock_parts/S = I
 		if(istype(S, /obj/item/weapon/stock_parts/manipulator))
```

Comment:
Makes swapping parts actually eject the old part/downgrade it. Reason is before all the old parts you stuck in it were just staying in its contents without being moved out or deleted, just no longer used.